includes/batch.inc

```diff
@@ -250,6 +265,38 @@ function islandora_paged_content_create_ocr_batch_operation(array $pages, array
 }
 
 /**
+ * Batch operation for consolidating OCR.
+ *
+ * @param string $paged_content_id
+ *   The paged content identifier to store the OCR.
+ * @param array $context
+ *   The context of this batch operation.
+ */
+function islandora_paged_content_create_ocr_paged_content_batch_operation($paged_content_id, &$context) {
+  module_load_include('inc', 'islandora_paged_content', 'includes/utilities');
+  $paged_content = islandora_object_load($paged_content_id);
+  if (empty($context['results']['pages'])) {
+    return;
+  }
+  islandora_paged_content_start_batch_operation($context['results']['pages'], $context);
```

Comment:
If using this wrapper stuff, should probably use the return of this, instead of mucking with the `$files` manually.